core/src/impl/Kokkos_DesulAtomicsConfig.hpp

```diff
@@ -4,9 +4,11 @@
 #ifndef KOKKOS_DESUL_ATOMICS_CONFIG_HPP
 #define KOKKOS_DESUL_ATOMICS_CONFIG_HPP
 
+#include <Kokkos_Macros.hpp>
 #include <impl/Kokkos_NvidiaGpuArchitectures.hpp>
```

Comment:
`Kokkos_Macros.hpp` includes `<impl/Kokkos_NvidiaGpuArchitectures.hpp>` so we coould even get rid of this include but I don't care much either way.